hydra-node/src/Hydra/HeadLogic.hs

```diff
@@ -525,4 +720,4 @@ rollback depth
       OpenState{previousRecoverableState} ->
         previousRecoverableState
       ClosedState{previousRecoverableState} ->
-        previousRecoverableState
+        previousRecoverableState
```

Comment:
Your editor is removing final newlines?

Reply:
I don't think so.
It was already there: [link](https://github.com/input-output-hk/hydra-poc/blob/master/hydra-node/src/Hydra/HeadLogic.hs#L528)
will add new line.